docs/manuals/core/config/task-config.md

```diff
@@ -7,7 +7,9 @@ A task configuration is necessary to instantiate a [Task](../concepts/task.md).
 - _**inputs**_: The input data nodes referring to the _function_'s parameter(s) data to be executed.
 - _**outputs**_: The output data nodes referring to the result(s) data of the _function_ to be executed.
 - _**skippable**_: Boolean attribute indicating if the task execution can be skipped if all output
-  data nodes are up-to-date. Default value: `False`.
+  data nodes are up-to-date (see *validity_period* attribute in the
+  [Data node management page](../entities/data-node-mgt.md) for more detail). The default value of
```

Comment:
Since we want to add the validity_period into the DataNodeConfig, I believe we should refer to `data-node-config.md` file, and add the same explanation as the one in entities/data-node-mgt.md. 
What do you think?

Reply:
This will be done in a separate PR on `validity_period` on DataNodeConfig